yarn.lock

```diff
@@ -372,8 +372,8 @@ error-ex@^1.2.0:
 
 es-abstract@^1.11.0:
   version "1.13.0"
-  resolved "https://artifactory.autozone.com:443/artifactory/api/npm/npm/es-abstract/-/es-abstract-1.13.0.tgz#ac86145fdd5099d8dd49558ccba2eaf9b88e24e9"
```

Comment:
Oh snap! Thanks! Didn't realize this made it out.

Reply:
It did because Travis had some issues with actually boot up on this project 😄